.github/workflows/validate-pull-request.yml

```diff
@@ -23,10 +23,10 @@ jobs:
     steps:
       - name: Add Jira Issue Code to Pull Request
         if: github.event_name == 'pull_request' && github.event.action == 'opened' 
-        uses: Lombiq/GitHub-Actions/.github/actions/add-jira-issue-code-to-pull-request@dev
+        uses: Lombiq/GitHub-Actions/.github/actions/Add-JiraIssueCodeToPullRequest-and-assign-issue-to-author@issue/OSOE-503
```

Comment:
GitHub Action names have to follow a kebab-case format, so all lower case with dashes separating the words.
Same below at _validate-submodule-pull-request.yml_.